examples/guides/mp-restful-webservice/src/main/docs/index.adoc

```diff
@@ -0,0 +1,487 @@
+[source,java]
+----
+include::{health-resource-class}[tag=javaxImports]
+----
+
+. Declare the class with these annotations:
++
+[source,java]
+----
+include::{health-resource-class}[tag=classDecl]
+----
+
+. Declare an injected instance field to refer to the greeting message resource. This
+is the only input to the activeness check in our simple implementation.
++
+[source,java]
+----
+include::{health-resource-class}[tag=greetingDecl]
+----
+<1> Indicates that JAX-RS should inject the field.
+<2> Because `GreetingMessage` is an application-scoped bean, JAX-RS will inject a
+reference to the single instance every time it creates this request-scoped resource.
+
+. Add the `checkHealth` method which decides whether the greeting resource is active
+(health) or not.
++
+[source,java]
+----
+include::{health-resource-class}[tag=checkHealthMethod]
+----
+
+. Add the method to implement the `/alive` endpoint
++
+[source,java]
+----
+include::{health-resource-class}[tag=aliveMethod]
+----
+<1> Implements the path `/alive`.
+<2> It's an HTTP `GET`.
+<3> Gets a `String` describing why the resource should _not_ be considered alive;
+null if the resource is OK.
+<4> Build the `Response` object for return accordingly.
+
+. Add the method to implement `/ready`
++
+[source,java]
+----
+include::{health-resource-class}[tag=readyMethod]
+----
+
+=== Update `GreetApplication`
+In the `getClasses` method add `HealthResource.class` to the arguments
+passed to `CollectionsHelper.setOf`. 
+
+include::{se-guide-adoc}[tags=rebuildAndRerunService;tryReadiness;tryLiveness]
+
+== Add Metrics Support
+include::{se-guide-adoc}[tag=metricsIntro]
+
+. Add the metrics dependency to `pom.xml`
++
+[source,xml]
+----
+include::{pom}[tag=metricsDependency]
+----
+
+. Annotate each method to be measured, in our case `getDefaultMessage`, 
+`getMessage`, and `updateGreeting`. (We annotate `updateGreeting` for simplicity
+and so the metrics
+reported here have the same values as for the Helidon SE RESTful web 
+service example. In a real application we might measure the `update` method separately
+from the `get` methods.)
++
+Your or your IDE will add these imports:
++
+[source,java]
+----
+include::{greet-resource-class}[tag=metricsImports]
+----
++
+[source,java]
+----
+include::{greet-resource-class}[tag=countedAnno]
+----
+<1> Marks this method as measured by a `Counter` metric.
+<2> Declares the unique name for this counter among all metrics.
+<3> Allows the same counter to accumulate uses of multiple methods.
+<4> Indicates that the metrics system should increment the counter on each invocation but
+_not_ decrement it when the method returns. 
+
+=== Rebuild and rerun your application
+Follow the same steps as before, remembering to stop any instance of your application
+that is still running.
+
+include::{se-guide-adoc}[tag=sendRequestsForMetrics]
+
+=== Send some requests
+Use the same `curl` commands from the beginning of the guide to send requests to
```

Comment:
CONTENT:  "same `curl` commands from the beginning of the guide " --> At the beginning, we provide only the HTTP calls, not the curl commands. 

So I suggest: "Use the following `curl` commands to send requests to the service:"